src/prefect/cli/auth.py

```diff
@@ -195,19 +208,20 @@ def switch_tenants(id, slug):
 
 @auth.command(hidden=True)
 @click.option("--name", "-n", required=True, help="A token name.", hidden=True)
-@click.option("--scope", "-s", required=True, help="A token scopre.", hidden=True)
-def create_token(name, scope):
+@click.option("--user-id", "-u", required=True, help="a user id.", hidden=True)
+def create_key(name, user_id):
     """
-    Create a Prefect Cloud API token.
-
-    For more info on API tokens visit https://docs.prefect.io/orchestration/concepts/api.html
+    Create a Prefect Cloud API Key for a user (must be active user or,
+    if active user is an Administrator, a Service Account in the active tenant).
```

Comment:
This active user blurb is confusing to me

Reply:
we use "active tenant" as vocabulary in client settings so I went with this "active user" terminology, but open to suggestions here for sure.  This docstring makes very little sense, even though it's technically accurate.  Will try to come back to it